content/news/044/index.md

```diff
@@ -66,6 +66,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Announcements
 
+### Rust GameDev Meetup
+
+![Gamedev meetup poster](gamedev-meetup.png)
+
+The 25th Rust Gamedev Meetup took place in March. You can watch the recording
```

Comment:
One copy and paste typo:
```suggestion
The 26th Rust Gamedev Meetup took place in March. You can watch the recording
```

Reply:
Ah, actually, maybe it's the link that's wrong - it points at the April meetup, which is #26, but I think it should be the March meetup, which is indeed #25 as you've put here

Reply:
Thanks for double checking, but I think it should be the 25th :sweat_smile: https://gamedev.rs/news/043/#rust-gamedev-meetup

In the case you counted from when it started, there was one month I missed :P

Reply:
Ooooh I see my mistake, thanks for pointing that out. I grabbed the wrong vid :100:

Reply:
Fixed in a new commit :+1: